config_script.py

```diff
@@ -45,11 +47,42 @@ def convert_object(args: list) -> None:
         sys.exit()
 
 
+def validate_args(parser: argparse.ArgumentParser,
+                  args: argparse.Namespace) -> None:
+    """Validate paths of config file and master dir."""
+    if os.path.isfile(args.config_path) and os.path.isdir(args.master_dir):
+        return
+    parser.print_usage()
+    print(f'{parser.prog}: error: Incorrect file path or directory')
```

Comment:
Here it would be good to print some more details, like (a) which value is invalid, and (b) what the value is.

Reply:
Gotcha, will write the function a little differently to incorporate that